common/src/main/java/com/wynntils/mc/utils/ComponentUtils.java

```diff
@@ -19,10 +19,10 @@
 import org.jetbrains.annotations.Nullable;
 
 public class ComponentUtils {
-    public static String getFormatted(FormattedText component) {
+    public static String getFormatted(FormattedText formattedText) {
```

Comment:
These method names are becoming a bit confusing with all my `FormattedText` usage. I might go with `getStringFormatted` and `getStringUnformatted`.

Reply:
I was thinking, we really need a clear name -- and possibly a separate type -- for "string with format codes", like "§eHello". Calling it "formatted" is clearly bad, since it gets confused with Vanilla's type system. Perhaps "coded string"? Or "fmtcode string"? 

This is really a different issue from this PR, but it aggravates the problem and makes it more urgent to solve. But I've been thinking a bit about this for some time before. I think we should consider introducing a separate type for CodedString (of whatever we end calling it), which basically just has a String as a field. But the point is that we can use Java's type system to make sure we handle strings properly. Then we can convert from a Component to a CodedString, or we can strip the control formatting from the CodedString and just return a bare String, etc. But at all points it will be clear if we're dealing with a CodedString or a Component. (And we should never really deal with a bare String, unless we're consuming it.)

Reply:
I noticed that in the class `Character` `getUnformatted` is used, but still checks for §-codes.
*snippet from Character:*
```java
ComponentUtils.getUnformatted(e.getTitle()).equals("§8§lSelect a Class"))
```
This is probably because Wynncraft uses §-coded strings for these container names. So this is not actually "unformatted" or "raw" or whatever we're gonna call it. I think we should do some actual string replacement to get rid of the §-codes in `getUnformatted` instead of just hoping that FormattedString::getString (or Component::getString) doesn't give us §-coded text.
*(Added a TODO to getUnformatted)*

Reply:
Whatever we need to do likely merits a separate PR, and perhaps some discussions before that. So let's do this change for now, and then we can come back and revisit it when we've decided how to best handle this.